### PR TITLE
Ctptest1

### PR DESCRIPTION
**Pull request by mbombara**
Task for reading CTP readout - publishing 3 histograms (BCID, Inputs and Classes).

### CODE REVIEW

File: Modules/CTP/src/RawDataQcTask.cxx [hunk 1]
```diff
@@ -46,6 +46,8 @@ void CTPRawDataReaderTask::initialize(o2::framework::InitContext& /*ctx*/)
   getObjectsManager()->startPublishing(mHistoBC);
   getObjectsManager()->startPublishing(mHistoInputs);
   getObjectsManager()->startPublishing(mHistoClasses);
+  mDecoder.setDoLumi(1);
```
Comment (Barthelemy) [inline Modules/CTP/src/RawDataQcTask.cxx:49]: These two methods take a boolean, please don't use `0` and `1` but use a boolean.

**Review by Barthelemy — APPROVED** [review]
A very minor comment otherwise it is good.

File: Modules/CTP/src/RawDataQcTask.cxx (hunk 1 above)
Comment (mbombara) [reply Modules/CTP/src/RawDataQcTask.cxx:49]: Ok, thanks! Should I change it now, or in the next release is ok? :-)

Comment (Barthelemy) [reply Modules/CTP/src/RawDataQcTask.cxx:49]: next release is ok

**Pull request merged**